ci-operator/step-registry/openshift-extended/logging-test/disruptive/openshift-extended-logging-test-disruptive-ref.metadata.json

```diff
@@ -2,18 +2,15 @@
 	"path": "openshift-extended/logging-test/disruptive/openshift-extended-logging-test-disruptive-ref.yaml",
 	"owners": {
 		"approvers": [
-			"IshwarKanse",
 			"anpingli",
 			"QiaolingTang",
-			"kabirbhartiRH",
-			"gkarager"
+			"kabirbhartiRH"
 		],
 		"reviewers": [
-			"IshwarKanse",
 			"anpingli",
 			"QiaolingTang",
-			"kabirbhartiRH",
-			"gkarager"
+			"kabirbhartiRH"
 		]
 	}
-}
\ No newline at end of file
+}
+
```

Comment:
Please don't add new line at the end of `*.metadata.json`

Reply:
Fixed.
```
1) check if there is newline at the end. cat -e <file>
2) remove the newline at the end of the file  truncate -s -1 file
```